### PR TITLE
Class03/exercises

### PR DESCRIPTION
**Pull request by kikobr82**
# Description
Class 03 exercises adjustments and content move from class05 to class03

> A brief explanation of what has been done.

## Type of change

- [ ] Exercise Submission (one exercise per PR)
  - [ ] This PR is just for **one** exercise of a class
  - [ ] PR name follows the pattern `<github-username>/<exercise-number>`
  - [ ] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [ ] Review [exercise submission steps](./README.md#Exercises)
- [X] Class content (instructors/admins)
- [ ] Documentation update
- [X] Repo management

### CODE REVIEW

File: classes/03class/exercises/c03-make01/ANSWER.md [hunk 1]
```diff
@@ -25,4 +25,4 @@ Insert command and output here
 <!-- Don't change anything below this point-->
 <!-- Before commiting, remove both commented lines--> 
 ***
-Answer for exercise [c05-make01](https://github.com/devopsacademyau/academy/blob/c41e824fb2a2c55e3a30b2371a87e3a7551b6741/classes/05class/exercises/c05-make01/README.md)
\ No newline at end of file
+Answer for exercise [c03-make01](https://github.com/devopsacademyau/academy/blob/779217898c5e63d5f3ca13cba3ac877e03f63efc/classes/03class/exercises/c03-make01/README.md)
```
Comment (denstorti) [inline classes/03class/exercises/c03-make01/ANSWER.md:28]: Update exercise title in README.md and Change commit SHA

File: classes/03class/exercises/c03-compose01/ANSWER.md [hunk 2]
```diff
@@ -20,4 +20,4 @@ Insert command and output here
 <!-- Don't change anything below this point-->
 <!-- Before commiting, remove both commented lines--> 
 ***
-Answer for exercise [c05-compose01](https://github.com/devopsacademyau/academy/blob/c41e824fb2a2c55e3a30b2371a87e3a7551b6741/classes/05class/exercises/c05-compose01/README.md)
\ No newline at end of file
+Answer for exercise [c03-compose01](https://github.com/devopsacademyau/academy/blob/779217898c5e63d5f3ca13cba3ac877e03f63efc/classes/03class/exercises/c03-compose01/README.md)
```
Comment (denstorti) [inline classes/03class/exercises/c03-compose01/ANSWER.md:23]: Update exercise title in README.md and update commit SHA

File: classes/03class/README.md [hunk 3]
```diff
@@ -4,3 +4,5 @@
 
  - [AWS ASG/Load Balancer/Route53](/classes/03class/aws/README.md)
```
[the remark this answers is not in the document]
Comment (kikobr82) [reply classes/03class/README.md:5]: adjusted

File: classes/03class/exercises/c03-docker07-deprecated/ANSWER.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Commands used to execute the image and the output of container execution:
+```
+Add command here
+```
+
+<!-- Don't change anything below this point-->
+<!-- Before commiting, remove both commented lines--> 
+***
+Answer for exercise [c03-docker06](https://github.com/devopsacademyau/academy/blob/c41e824fb2a2c55e3a30b2371a87e3a7551b6741/classes/04class/exercises/c04-docker06/README.md)
```
[the remark this answers is not in the document]
Comment (kikobr82) [reply classes/03class/exercises/c03-docker07-deprecated/ANSWER.md:20]: adjusted

File: classes/03class/exercises/c03-compose01/ANSWER.md (hunk 2 above)
Comment (kikobr82) [reply classes/03class/exercises/c03-compose01/ANSWER.md:23]: Done

**Pull request merged**